spec/GraphQLOverHTTP.md

```diff
@@ -615,6 +615,18 @@ If
 raises a _GraphQL request error_, the server SHOULD NOT execute the request and
 SHOULD return a status code of `200` (Okay).
 
+For example the well-formed GraphQL-over-HTTP request:
+
+```json
+{
+  "query": "query getItem($id: String!) { item(id: $id) { id } }",
```

Comment:
> I think it is more common to use PascalCase for operation names

Perhaps; but I usually see schemas that follow this: https://www.apollographql.com/docs/technotes/TN0002-schema-naming-conventions/

> I also think it is best practice to use the ID type for identifiers.

Agree

Reply:
I actually went through the spec and extracted the conventions from that; results are here:

https://benjie.dev/graphql/naming#operation-names

The spec uses a mixture for operation names but camelCase is more common than UpperCamelCase. You may find the part on fragments interesting because I had a conversation with Lee about it.

Reply:
I tweaked your example and committed that, I added a `get` prefix to match spec examples such as `getDogName` https://spec.graphql.org/draft/#example-069e1